### ISSUE TITLE
Generate questions for solution-only exams

### ISSUE THREAD

**Issue by wace-vault** [wace-vault/collection#9]
Currently there are a couple exam and test papers which are marked as (SOLS ONLY), meaning that we only have solutions for the tests and not the original test paper. For the majority of these tests, question-papers could be made by copying the answer-paper and hiding all the answers.
Although a minor issue, help would be appreciated.

**Comment by chromo4** [comment]
can you please assign with an issue to work on and some guidance on how to get started? Thank yo

**Comment by wace-vault** [comment]
> can you please assign with an issue to work on and some guidance on how to get started? Thank yo

The issue should say everything you need to know to get started.

**Comment by jtruong05** [comment]
could I get the link to the discord server?

**Comment by wace-vault** [comment]
> could I get the link to the discord server?

No, sorry. https://github.com/wace-vault/collection/issues/3#issuecomment-1472152727
